XML/TestCases/FunctionalTests-WALA.xml

```diff
@@ -128,4 +128,18 @@
         <Tags>wala</Tags>
         <Priority>2</Priority>
     </test>
+    <test>
+        <TestName>WALA-VERIFY-HOSTNAME-CHANGE</TestName>
+        <testScript>WALA-VERIFY-HOSTNAME-CHANGE.py</testScript>
```

Comment:
This python script has many lines in common with WALA-VERIFY-HOSTNAME.py. Can we use a single script with two different parameters?

Reply:
sure, thanks Ju, I'd like move the common functions to the library file.

Reply:
If this is ready for the review, please post the label 'Ready-For-Review'

Reply:
@juhlee-microsoft  I double check, only CheckHostName function is duplicated, and It is not suitable to move this into common library.